api/api/src/test/scala/uk/ac/wellcome/platform/api/ApiSearchTemplatesTest.scala

```diff
@@ -0,0 +1,34 @@
+package uk.ac.wellcome.platform.api
+
+import akka.http.scaladsl.model.ContentTypes
+import io.circe.Json
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.platform.api.works.ApiWorksTestBase
+
+class ApiSearchTemplatesTest
+    extends ApiWorksTestBase
+    with Matchers
+    with JsonHelpers {
+
+  it("renders a list of available search templates") {
+    checkJson { json =>
+      json.isObject shouldBe true
+      val templates = getKey(json, "templates")
+      templates.isEmpty shouldBe false
```

Comment:
Slightly more explicit:

```suggestion
      templates should not be empty
```